### PR TITLE
Mattermost notifications with Webhook/Matterhook

### PR DESCRIPTION
**Pull request by Ramel**
**Problem**
- Send Kitsu notifications to Mattermost (Tested on Mattermost Version: 5.37.2)

**Solution**
- Need also to update Zou to same build;
- Perhaps a rebase error with "src/locales/en.js" file, which was modified after. Line 699 notifications_slack_user: 'Slack member ID',

Mattermost setup
=================
Need to create in Mattermost a User, for example **"Kitsu"**, as **"Team Admin"** (so it can set a Webhook).  
Need to create a Channel for the notifications, for example **"Kitsu"**.
Connect in Mattermost with this user account **"Kitsu"**. Go to **"Integrations"** / **"Incoming Webhooks"** / **"Add incoming Webhook"** with :
- Title: Kitsu
- Channel: Kitsu (created before)
- Username: Kitsu

Then copy the URL generated to Kistu web interface, in **Organisation** settings view, Integrations and fill **"Mattermost Webhook (Optional)"** fields with it.

Then, each user can set in his profile, the **"Mattermost notifications Enabled"** field to "Yes" and enter his **"Mattermost Username"** to receive Kitsu Notifications into Mattermost.

### CODE REVIEW

**Comment by frankrousseau** [comment]
Hello @Ramel,

Thank you for this contributiion. Is it ok for us to merge this PR?

**Comment by Ramel** [comment]
As it's my first Pull request here, I hope it's ok!
My only concern is how the Flask migration is done in final delivery as we have added columns in the db.
But the code for me si ok and tested our own server!

File: .eslintrc.js [hunk 1]
```diff
@@ -17,7 +17,8 @@ module.exports = {
     'jest/no-focused-tests': 'error',
     'jest/no-identical-title': 'error',
     'jest/prefer-to-have-length': 'warn',
-    'jest/valid-expect': 'error'
+    'jest/valid-expect': 'error',
+    'indent': 'off'
```
Comment (frankrousseau) [inline .eslintrc.js:21]: Why did you remove this? Indentation should follow a strict scheme.

**Comment by frankrousseau** [comment]
Thank you a lot for this contribution! It will improve the life of other studios for sure!

File: .eslintrc.js (hunk 1 above)
Comment (Ramel) [reply .eslintrc.js:21]: We only have added the `'indent': 'off'`, if not set the compilation hangs.
But it's perhaps a thing to keep in mind! 
Perhaps specific to our Dev config? Don't known if we should have keep it in the final commit.

Comment (frankrousseau) [reply .eslintrc.js:21]: I removed it. The compilation hangs because it waits that you fix the indentation of the mentioned files. It's here to make sure the code stays similar everywhere :).
